components/teams-service/storage/postgres/postgres.go

```diff
@@ -16,20 +16,10 @@ import (
 	"github.com/chef/automate/lib/db"
 	"github.com/chef/automate/lib/grpc/auth_context"
 	"github.com/chef/automate/lib/logger"
-	uuid "github.com/chef/automate/lib/uuid4"
 )
 
 var emptyOrWhitespaceOnlyRE = regexp.MustCompile(`^\s*$`)
 
-// WARNING
-// TODO (tc): The storage interface is still using V1 verbiage, so
-// name is really the ID in V2 terms. We'll refactor at GA when V1 is removed.
-// Also, project filtering has only been implemented for storage functions
-// that are used by the V2 server.
-
-// TODOs
-// - CREATE EXTENSION "uuid-ossp"; (currently done via Makefile for tests)
```

Comment:
Hah! There's a line in the `Makefile` to remove that'll bring this PR to `-2000` 🎉